### PR TITLE
Initial hugging face model

### PR DESCRIPTION
**Pull request by EvilDrPurple**
#### Fixes

- Part of #12 

#### Description

- Adds Hugging Face training model testing script for URL classification

### CODE REVIEW

**Comment by josh-chamberlain** [comment]
nice, I think with a readme explaining how this relates to huggingface / how to update the training data, this is a good start.

File: hf_testing/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+You can run the training script by installing the libraries in `requirements.txt` and running `python3 huggingface_test.py`. Everythning should be setup and ready to go. You can change the training model by editing the `MODEL` variable and number of training steps with the `MAX_STEPS` variable.
+
+### Using the trained model
+
+You can test the trained model by entering one link at a time in the Inference API GUI box on the model's [homepage](https://huggingface.co/PDAP/url-classifier-test).
+
+You can also consume the trained model programmatically using Hugging Face's Inference API. Learn more about that here: [https://huggingface.co/docs/api-inference/quicktour](https://huggingface.co/docs/api-inference/quicktour)
+
+## TODO
```
Comment (josh-chamberlain) [inline hf_testing/README.md:60]: these are really great, could you please make issues for them? you don't have to flesh them out, but I'd rather have issues than a readme to keep up to date

**Comment by josh-chamberlain** [comment]
@mbodeantor this passes my checks 👀

**Review by mbodeantor — APPROVED** [review]
@EvilDrPurple Nice! For future work, might be worth looking at the distribution of labels to see if there are some very infrequent ones that might be worth excluding to boost the accuracy. Also might be worth more strategic sampling to make sure every group is equally or proportionally represented in the training data. Finally, I built my own desktop with a pretty hefty GPU thinking I would use it for ML eventually if there is a way to GPU enable models to speed up training.

**Pull request merged**